modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostgisNGDataStoreFactoryTest.java

```diff
@@ -38,11 +38,13 @@
 import org.geotools.jdbc.SQLDialect;
 import org.geotools.util.Version;
 import org.junit.After;
+import org.junit.Assert;
```

Comment:
This file already has a static import for Assert.assertEquals. Why not just use that?

Reply:
whoops. removed this import.